src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -501,3 +501,104 @@ def _test_log_file_mgmt(self, resource_group, server, database_engine):
                               JMESPathCheck('type(@)', 'array')]).get_output_in_json()
 
         self.assertIsNotNone(result[0]['name'])
+
+
+class ReplicationMgmtScenarioTest(ScenarioTest):  # pylint: disable=too-few-public-methods
+
+    @ResourceGroupPreparer(parameter_name='resource_group')
+    def test_mysql_replica_mgmt(self, resource_group):
+        self._test_replica_mgmt(resource_group, 'mysql')
+
+    def _test_replica_mgmt(self, resource_group, database_engine):
+        server = self.create_random_name(SERVER_NAME_PREFIX, 32)
+        replicas = [self.create_random_name('azuredbclirep1', SERVER_NAME_MAX_LENGTH),
+                    self.create_random_name('azuredbclirep2', SERVER_NAME_MAX_LENGTH)]
+
+        # create a server
+        result = self.cmd('{} server create -g {} --name {} -l brazilsouth '
+                          '--admin-user cloudsa --admin-password SecretPassword123 '
+                          '--sku-name GP_Gen4_2'
+                          .format(database_engine, resource_group, server),
+                          checks=[
+                              JMESPathCheck('name', server),
+                              JMESPathCheck('resourceGroup', resource_group),
+                              JMESPathCheck('sslEnforcement', 'Enabled'),
+                              JMESPathCheck('sku.name', 'GP_Gen4_2'),
+                              JMESPathCheck('replicationRole', 'None'),
+                              JMESPathCheck('masterServerId', '')]).get_output_in_json()
+
+        from time import sleep
+        sleep(300)
```

Comment:
why the call of sleep()?

Reply:
This is a backend issue, if create a source server and then immediately create a replica for it, there may be no first backup, will cause the create replica failed. here sleep a while is to make sure the create replica action success when really run those cases.

Reply:
Okay, I understand. Although  shouldn't mysql server create block till the server is created?

If this is not the case, is it possible to poll the first server every 5s or so until it has been successfully created using the show command?

Reply:
The fist server was already created successfully. Here is to create a replica for the first server. It is by design, Even after the first server is ready, we also need wait a while to make sure the restore (create replica) from it can be successful.

Reply:
Okay, that's fair. Thanks for letting me know.